server/src/main/java/org/apache/druid/client/indexing/ClientCompactQueryTuningConfig.java

```diff
@@ -54,11 +59,15 @@ public static ClientCompactQueryTuningConfig from(
         userCompactionTaskQueryTuningConfig == null ? null : userCompactionTaskQueryTuningConfig.getMaxRowsInMemory(),
         userCompactionTaskQueryTuningConfig == null ? null : userCompactionTaskQueryTuningConfig.getMaxBytesInMemory(),
         userCompactionTaskQueryTuningConfig == null ? null : userCompactionTaskQueryTuningConfig.getMaxTotalRows(),
+        userCompactionTaskQueryTuningConfig == null ? null : userCompactionTaskQueryTuningConfig.getSplitHintSpec(),
```

Comment:
May be more readable if the check for null `userCompactionTaskQueryTuningConfig` is moved up:

```java
if (userCompactionTaskQueryTuningConfig == null) {
    return new ClientCompactQueryTuningConfig(
        maxRowsPerSegment,
        null,
        null,
        ...
    )
} else {
    return new ClientCompactQueryTuningConfig(
        maxRowsPerSegment,
        userCompactionTaskQueryTuningConfig.getMaxRowsInMemory(),
        userCompactionTaskQueryTuningConfig.getMaxBytesInMemory(),
        ...
    )

}
```

Reply:
Good point. Fixed.